### PR TITLE
Expand CSS section and add something on media queries

### PR DESCRIPTION
**Pull request by bertfrees**
Fixes https://github.com/daisy/ebraille/issues/189

@mattgarrish I need you to check that there's not too big of a break in style with what you have written so far, and also that I haven't written anything that a native English speaker wouldn't write.

@wfree-aph, @GeorgeKerscher, @clapierre, @avneeshsingh

* [Preview](https://raw.githack.com/daisy/ebraille/css/index.html)
* [Diff](https://services.w3.org/htmldiff?doc1=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://daisy.github.io/ebraille/index.html&doc2=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://raw.githack.com/daisy/ebraille/css/index.html)

### CODE REVIEW

**Comment by bertfrees** [comment]
There is at least one thing that I expect will need to be changed, namely that the reading system requirements (https://github.com/daisy/ebraille/pull/192/commits/7620d96888b3bab94f870abd018be5b974dd9f3e) should probably be moved to a separate section.

**Review by clapierre — APPROVED** [review]

**Comment by bertfrees** [comment]
My last commit provides the structure I have in mind for the CSS best practices. It is as follows:

- Media queries
- Spacing
- Indentation
- Alignment, positioning and spatial arrangement
- Text spacing
- White space handling
- Wrapping and breaking text
- Page breaking and column layout
- Borders and lines
- Rendering of tables
- Automatic numbering
- Floating content

[Preview](https://raw.githack.com/daisy/ebraille/css/best-practices/styling/index.html)

Next I will work on arranging the existing examples according to this structure. We also need more examples. Perhaps I should indicate in the document for which requirements we're still looking for examples, and then ask the group at the next meeting.

**Review by wfree-aph — COMMENTED** [review]
Would only suggest minor changes.

Line 876: add word "as" so it reads "regarded as unreliable"
Line 876-877- remove "be able to" as it is redundant

File: index.html [hunk 1]
```diff
@@ -63,6 +63,30 @@
 						"title": "Digital Publishing WAI-ARIA Module",
 						"href": "https://www.w3.org/TR/dpub-aria/",
 						"publisher": "W3C"
+					},
+					"css-page": {
+						"aliasOf": "css-page-3-20230914"
```
Comment (mattgarrish) [inline index.html:68]: It's best not to alias dated drafts like this as we'll be stuck with the dated version as they get updated. Is there a reason you can't use "css-page-3", for example, as those references always point to the latest published version.

**Review by mattgarrish — APPROVED** [review]
I've only read over the spec edits so far, but looks fine to me from a technological perspective. One comment about the aliasing of specifications, but otherwise just some minor edits I can make later (e.g., best to avoid "author" as it's caused nothing but confusion in epub with the actual content author and while I don't expect the same problems here it'd be good to keep our terminology aligned).

**Review by wfree-aph — APPROVED** [review]
Thanks for making those two minor changes! Looks great to me!

**Comment by bertfrees** [comment]
I added a request for more examples in the CSS best practices, and two other harmless commits just now. I will merge the PR before today's meeting.

Rearrangement of best practices document will come in a next PR.

**Pull request merged**